mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -246,6 +246,12 @@ static cl::opt<std::string> randomData(
         "use all ones. Otherwise, use time(0) as the seed."),
     cl::value_desc("seed"), cl::init("none"));
 
+static cl::opt<std::string>
+    randomSide("rand_side",
+               cl::desc("To populate random numbers to a specified tensor, "
+                        "e.g. -rand_side filter"),
+               cl::value_desc("tensor"), cl::init("both"));
```

Comment:
e2e test case using `both` is not provided.

Reply:
"both" is a default value. All existing random tests belong to this case.